### ISSUE TITLE
Bagnon 10.2.13 - Merged Servers Not Showing Cross-Realm Item Counts

### ISSUE THREAD

**Issue by voidzone** [Jaliborc/Bagnon#1858]
**Which software were you running?**
 - Addon version name: Bagnon 10.2.13
 - Client used: Retail

**Have you read the changelog?**
Yep

**Please describe the bug.**
On merged servers, addon is no longer showing item counts for characters on the other servers

**Please describe how to reproduce it.**
1. Login to a char on a merged server
2. Check the count on a common item
3. Observe the lack of details from characters from the other server(s)
4. Repeat with a character on one of the other servers to confirm
5. Profit?

**Error Logs are Important!**
No Errors

**Comment by bubfeeld** [comment]
so are you seeing this work correctly on servers that arent merged?

**Comment by voidzone** [comment]
Yeah, I'm seeing correct item counts for characters on the current server, just not the other 2 that are merged with it.
